docs/providers/documentation/auth0-provider.mdx

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Please mention that if the user is looking for Auth0 as an auth method for keep, they should refer to https://docs.keephq.dev/deployment/authentication/auth0-auth